hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/io/nativeio/NativeIO.java

```diff
@@ -819,6 +849,12 @@ public static boolean access(String path, AccessRight desiredAccess)
         }
       }
     }
+
+    public static void setConf(Configuration conf) {
```

Comment:
Should we name this a little bit more specific and not only setConf? initTransmitFileEnabledConf()?

Reply:
it's not meant to be only for enabling transmitfile, but generally for reloading/resetting the configuration